.aegir.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('aegir').PartialOptions} */
+export default {
+  dependencyCheck: {
+    ignore: [
+      'dotenv',
+      'typescript',
+      'wait-on',
```

Comment:
Are we sure these are used?  Can they be removed from the project if not?

Reply:
yep. they're used in npm scripts.